fusion/Modules/Lua/cryptomatte_utilities.lua

```diff
@@ -663,468 +524,387 @@ function is_position_in_rect(rect, x, y)
     return true
 end
 
-function get_absolute_position(image, relative_x, relative_y)
+function module._hex_to_float(hex)
```

Comment:
Isolated function for future fixes/improvements regarding issue https://github.com/Psyop/Cryptomatte/issues/100 (PR https://github.com/Psyop/Cryptomatte/pull/120)

Reply:
Fixed.

Issue: #14 
PR: #15